### DISCUSSION TITLE
Official android app

### DISCUSSION THREAD

**Discussion by rajil** [blakeblackshear/frigate#11242] [General]
Can we get an android app developed by the frigate team?

**Comment by NickM-27** [comment]
As a professional android developer, I see no need for this. The next version of frigate is much more mobile friendly and there aren't really any benefits to doing a native app over a PWA for frigates usecase

**Comment by rajil** [comment]
I dont see how to run PWA in Firefox mobile. 

For a well done android app for NVR look at https://github.com/ZoneMinder/zmNinja

**Reply by NickM-27** [reply]
pwa requires https and it is supported mainly in the new UI https://github.com/blakeblackshear/frigate/discussions/11136

as I said, I see nothing that frigate would need that could not be supported in a PWA
